src/NServiceBus.Persistence.AzureStorage.AcceptanceTests/ConfigureEndpointAzureStorageQueueTransport.cs

```diff
@@ -13,7 +13,7 @@ public class ConfigureEndpointAzureStorageQueueTransport : IConfigureEndpointTes
     {
         var connectionString = settings.Get<string>("Transport.ConnectionString");
         //connectionString = "UseDevelopmentStorage=true;";
-        configuration.UseSerialization<XmlSerializer>();
+        configuration.UseSerialization<JsonSerializer>();
```

Comment:
this line found a bug (or even more?) in ASQ. So adjusting it (you could also just remove it) just makes us blind to potential issues?